client/src/pages/demo/components/index.tsx

```diff
@@ -13,6 +13,7 @@ import Checkbox from '@/src/shared/components/Checkbox';
 import Select from '@/src/shared/components/Select';
 import Button from '@/src/shared/components/Button';
 import Radiobutton from '@/src/shared/components/Radiobutton';
+import Table from '@/src/shared/components/Table';
```

Comment:
- i noticed code all has weird spaces or tab. Could you try format this using prettier all file changes.